account_statement_import_move_line/models/account_move_line.py

```diff
@@ -32,6 +32,13 @@ def _prepare_statement_line_vals(self, statement):
         return vals
 
     def create_statement_line_from_move_line(self, statement):
+        # Check if the statement has a journal assigned to it and set a
+        # default journal. This can be modified in the form lines
+        if not statement.journal_id:
+            bank_journal = self.env["account.journal"].search(
+                [("type", "=", "bank")], limit=1
+            )
+            statement.journal_id = bank_journal
```

Comment:
I think this is not necessary because it will be defined automatically when a line is created; if you should specify in the `_prepare_statement_line_vals()` method the `journal_id` that corresponds.

Reply:
Being a computed field, you need to infer the journal other way. Include it in the wizard, and fill it depending on the context (I think it's included in the context when coming from the dashboard).

Reply:
I have decided to keep the current logic for cases where a statement with added lines already exists, in case more lines need to be added, limiting the visibility of the current button to the condition "not journal_id."

Additionally, I have added the option to import lines from the dashboard by creating a new statement.

If at some point you permanently lose the form view of the statement, you can continue with the new dashboard option.